src/components/ArticleCommentsList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../articles_api_utils";
+import { CommentCard } from "./CommentCard";
+import { FilterBar } from "./FilterBar";
+import { PageNavigator } from "./PageNavigator";
+import { Loading } from "./Loading";
+
+export const ArticleCommentsList = ({ articleId }) => {
+  const [commentsList, setCommentsList] = useState([]);
+  const [totalComments, setTotalComments] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+  const [limit, setLimit] = useState("10");
+  const [p, setP] = useState("1");
+  useEffect(() => {
+    setIsLoading(true);
+    getCommentsByArticleId(articleId, limit, p).then(({ comments }) => {
+      setCommentsList(comments);
+      setTotalComments(comments[0].total_results);
+      setIsLoading(false);
+    });
+  }, [articleId, limit, p]);
+
+  return (
+    <>
+      <FilterBar limit={limit} setLimit={setLimit} setP={setP} />
+      {isLoading ? (
+        <Loading />
+      ) : (
+        <div className="article-list-wrapper">
+          <ul className="article-list">
+            {commentsList.map((comment) => {
```

Comment:
Ideally we also want some logic, so that if there are no comments for an article yet, that we display some sort of user feedback. This may be a message stating 'no comments yet!' - have a look at your optional rendering for this.

Reply:
Will do! Also sorry about the images, I have a plan to apply styling once all my components have been created so I made them tiny to keep them out the way. Happy to make them bigger in the meantime?